### ISSUE TITLE
Add missing comma after the last element declaration in multi-line slice, array or map literal

### ISSUE THREAD

**Issue by dlsniper** [go-lang-plugin-org/go-lang-idea-plugin#2238]
```go
package main

func main() {
	x := []int{
		1,
		2
	}
	_ = x
}
```

should become

```go
package main

func main() {
	x := []int{
		1,
		2,
	}
	_ = x
}
```

**Comment by dlsniper** [comment]
Technically we already have the error here but I'm not sure if we can add quickfixes to the errors that come from the parser.

**Comment by grenki** [comment]
fixed in c13502f19a41ad4c8d48df5c912e12c5a688770e
